src/containers/NewPostForm/index.css

```diff
@@ -0,0 +1,15 @@
+.row {
```

Comment:
I would avoid these names like row and container, they might be used bu bootstrap or someone else and make a conflict, I would go with more unique names

Reply:
PTAL